src/Weasel.Postgresql/Tables/IndexDefinition.cs

```diff
@@ -436,20 +452,47 @@ private static string canonicizeColumn(string expression)
             return CanonicizeCast(expression);
         }
 
-        private static string removeSortOrderFromExpression(string expression, out SortOrder order)
+        private static string removeSortOrderFromExpression(string expression, out SortOrder order, out NullsSortOrder? nullsSort)
         {
             if (expression.EndsWith("DESC)"))
             {
                 order = SortOrder.Desc;
+                nullsSort = null;
                 return expression.Substring(0, expression.Length - 6) + ")";
             }
+            else if (expression.EndsWith("DESC NULLS FIRST)"))
+            {
+                order = SortOrder.Desc;
+                nullsSort = Tables.NullsSortOrder.First;
+                return expression.Substring(0, expression.Length - 18) + ")";
```

Comment:
**SUGGESTION:** I know that it was already like that, but it might be worth making `"DESC NULLS FIRST)"` a const, and using here its length instead of "magic" number (same for other places). It's not a blocker for me if you don't agree.

Reply:
Not really an issue if it's only used once. Kinda helpful to see the DB terminology in place here.

Reply:
Fixed and handled this suggestion.